### ISSUE TITLE
Make TTL configurable from the CLI

### ISSUE THREAD

**Issue by majoferenc** [terricain/certbot-dns-azure#48]
TTL entry is at the moment created with static value of 120 seconds, it would be better to have an option to configure the value via CLI argument like `ttl-txt-dns-record`.
Example:

    sudo certbot certonly \
    --authenticator dns-azure \
    --dns-azure-config azure.ini \
    --preferred-challenges dns-01 \
    --ttl-txt-dns-record 10 \
    --dns-azure-propagation-seconds 10 \
    --noninteractive \
    --server https://acme-v02.api.letsencrypt.org/directory \
    --key-type rsa -d "*.dummy.example.com" \
     -m email@example.com \
    --agree-tos \
    --preferred-chain "ISRG Root X1" \
    --force-renewal \
    -v

**Comment by terricain** [comment]
Good idea, I don't have time for doing it at the moment but would be happy to review a PR
